src/themes/custom/app/logout-page/logout-page.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container w-100 h-100">
```

Comment:
Same here. It's unclear to me why a `logout-page.component.html` is being added to the `custom` theme in this PR.  Is this accidental?